apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9084,9 +9103,53 @@ ngx-daterangepicker-material {
         }
     }
 }
+/*notofication css*/
+.HW_badge.HW_visible{
```

Comment:
Avoid using improper names in class names (Applicable to all class names that begin with HW)

Reply:
This is third party classes , we can't change it and can't add new classes in this